src/plugins/rv-packages/otio_reader/rv_demo.json

```diff
@@ -0,0 +1,22 @@
+{
+	"entities": {
+		"rv_asset1": {
+			"versions": [
+				{
+					"traits": {
+						"locatableContent": { "url": "/Users/nelsonr/test_media/BBB_08_a-team_014_ANIM_001.mp4" }
+					}
+				}
+			]
+		},
+		"rv_asset2": {
+			"versions": [
+				{
+					"traits": {
+						"locatableContent": { "url": "/Users/nelsonr/test_media/BBB_08_a-team_014_ANIM_002.mp4" }
```

Comment:
n.b The MediaCreation trait ID ends up being `openassetio-mediacreation:content.LocatableContent`